apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/config/GrpcConfig.java

```diff
@@ -29,4 +34,25 @@ public interface GrpcConfig {
   /** @return Optional deadline duration for the each RPC to the bridge. Defaults to 30 seconds. */
   @WithDefault("PT30S")
   Optional<Duration> callDeadline();
+
+  /** @return Defines retry strategy for bridge calls when using {@link RetriableStargateBridge}. */
+  @Valid
+  @NotNull
+  Retries retries();
+
+  interface Retries {
+
+    /** @return If call retries are enabled. */
+    @WithDefault("true")
```

Comment:
We could alternatively just use `maxAttempts = 0` to indicate disabled (I am ok with separate `enabled` too).

Reply:
I like explicit boolean and also a validation that max retries is positive (which I miss, so adding)